Library/Formula/wine.rb

```diff
@@ -5,7 +5,6 @@
 class Wine < Formula
   desc "Wine Is Not an Emulator"
   homepage "https://www.winehq.org/"
-  revision 1
```

Comment:
We only added it because the patch was removed when the devel version was bumped last time.